storage/tianmu/handler/ha_tianmu.cpp

```diff
@@ -1634,7 +1634,13 @@ bool ha_tianmu::inplace_alter_table(TABLE *altered_table, Alter_inplace_info *ha
         DBUG_RETURN(false);
     } else if (!(ha_alter_info->handler_flags & ~TIANMU_SUPPORTED_ALTER_ADD_DROP_ORDER)) {
       std::vector<Field *> v_old(table_share->field, table_share->field + table_share->fields);
-      std::vector<Field *> v_new(altered_table->s->field, altered_table->s->field + altered_table->s->fields);
+
+      Field_iterator_table it_field;
```

Comment:
Because i want to get the default value from the filed, but altered_table->s->field's table pointer is null, can not to get the default value.  alter_table->field's table pointer is correct, so i use it to store the fields;